subsys/bluetooth/services/fast_pair/fp_storage/include_priv/fp_storage_ak_priv.h

```diff
@@ -46,6 +46,30 @@ extern "C" {
 #define SETTINGS_AK_NAME_MAX_SIZE \
 	(sizeof(SETTINGS_AK_FULL_PREFIX) + SETTINGS_AK_NAME_MAX_SUFFIX_LEN)
 
+/** Settings key name prefix for bond info. */
+#define SETTINGS_BOND_NAME_PREFIX "bond_info"
```

Comment:
Consider naming the defines as BOND_INFO for consistency

Reply:
I wonder if it should actually be `bond_info` or just `bond` (the `bond` term could refer to the data stored in NVM, `bond_info` is the NVM data + RAM data that is used during store handling)

Reply:
After the rename the struct is also named bond_info